torch/utils/_debug_mode.py

```diff
@@ -204,9 +204,12 @@ def hash_tensor_fn(
     else:
         t_clean = t.to(dtype=torch.int64)
 
-    out = torch.hash_tensor(t_clean)
-    if use_scalar:
-        return out.item()  # type: ignore[attribute]
+    out = 0
```

Comment:
should you return either Tensor(0) or 0 depending on use_scalar, to make things uniform between 0- and 1+-element tensors?

Reply:
good catch, thx! fixed